### ISSUE TITLE
Investigate unstable convergence in Burgers' training example

### ISSUE THREAD

**Issue by christianfenton** [christianfenton/jax-fno#1]
The convergence of loss function during training of the FNO seems to be unstable, with sharp spikes occuring frequently. 
This points to:
1. problems in the implementation of the FNO itself
2. problems in the implementation of the training routine
3. problems with the choice of hyperparameters.

To resolve this issue: 
- [ ] Write unit tests for each component in (1.) and (2.)
- [ ] Vary the hyperparameters systematically and study the resulting performance of the model
